lib/main/tsconfig/tsconfig.ts

```diff
@@ -574,6 +574,7 @@ function getDefinitionsForNodeModules(projectDir: string, files: string[]): { ou
             if (fs.existsSync(file + '.d.ts')) {
                 return file + '.d.ts';
             }
+            return file;
         });
```

Comment:
`return undefined` and then do a `.filter(x=>!!x)`

PS: Sorry about the irregular whitespace. Thanks for fixing that :rose: